donation-api/src/main/java/org/interfaithsanctuary/donationapi/controller/DonationController.java

```diff
@@ -0,0 +1,32 @@
+package org.interfaithsanctuary.donationapi.controller;
+
+
+import org.interfaithsanctuary.donationapi.model.CalloutNeed;
+import org.interfaithsanctuary.donationapi.model.Donation;
+import org.interfaithsanctuary.donationapi.repository.CalloutNeedRepository;
+import org.interfaithsanctuary.donationapi.repository.DonationRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@RestController

```

Comment:
May need to enable cross origin or configure the cross origin settings so that this controller won't throw an error when the API is used.

Reply:
I haven't configured that but it doesn't seem to throw any errors when posting to the specified URL. Haven't tried to use it from the front-end, you think it's alright?